server.js

```diff
@@ -29,4 +30,20 @@ app.get('/api/films', (req, res) => {
   });
 });
 
+app.get('/api/sessions/:film', (req, res) => {
+  Film.find({ slugName: req.params.film }, function(err, docs) {
+    if (err) return res.status(500).json({ message: 'Запрос не выполнен' });
+    const filmName = docs[0].name;
+
+    const id = docs[0]._id;
+
+    Session.find({ filmId: id }, function(err, docs) {
```

Comment:
Can you try to do it in one db request? You might find the following link useful https://mongoosejs.com/docs/populate.html. Or maybe some other approach.